text/0006-cocoa-sdk-swift-demangle.md

```diff
@@ -0,0 +1,33 @@
+* Start Date: 2022-09-08
+* RFC Type: decision
+* RFC PR: https://github.com/getsentry/sentry-cocoa/pull/2033
+
+# Summary
+
+This RFC proposes the inclusion of Swift demangle logic in the cocoa SDK, 
+the same way symbolicator do.
+
+# Motivation
+
+We use class names as transaction names and transaction screen data, and the name
+is not user friendly if we don't demangled it.
+
+# Drawbacks
+
+- By copying the demangle logic from Swift open source project, 
+we increate the SDK binary by 46% (from 3.3MB to 4MB).
```

Comment:
Is it really 4 MB? The % doesn't match:

3.3 MB + 46% = 4.8 MB
3.3 MB + 24% = 4 MB

```suggestion
we increate the SDK binary by 24% (from 3.3MB to 4MB).
```

Reply:
Your math is right, my description is wrong. from 3.0mb to 4.4mb.